Taarafo.Core/Services/Foundations/Posts/PostService.Exceptions.cs

```diff
@@ -34,6 +34,10 @@ private async ValueTask<Post> TryCatch(ReturningPostFunction returningPostFuncti
             {
                 throw CreateAndLogValidationException(invalidPostException);
             }
+            catch (NotFoundPostException notFoundPostException)
+            {
+                throw CreateAndLogValidationException(notFoundPostException);
```

Comment:
Wrap the exception to the local exception here and then call the method with the local model.